src/core/data/all_script_names.hpp

```diff
@@ -3,9 +3,9 @@
 namespace big
 {
 	constexpr const static inline auto launcher_scripts = std::to_array({
-	    0u,
```

Comment:
Could this be refactored into an initializer_list? https://en.cppreference.com/w/cpp/utility/initializer_list That way, it would allow the compiler to optimize this out when references occur. Use the iterator functions/helpers to iterate it.

Reply:
initializer_list might be implemented as just a pointer + len (array like) but might aswell just be safe and use king std::array